### DISCUSSION TITLE
How to hide a private repository from everyone's profile page?

### DISCUSSION THREAD

**Discussion by froitag** [community/community#61612] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi,

I'm wondering if it is possible to hide a private repository from every contributor's profile page?

I know that as a contributor I can hide private projects from my own profile page. 

But I'm wondering if I as a private repository owner can configure this globally for all my contributors?

Right now potentially critical information like PR names are leaking to the profile pages of my contributors which I'd like to prevent.

Thanks for your help guys!

**Comment by nirav-gajera** [comment, marked as answer]
there is no way to hide a private repository from every contributor's profile page. As you mentioned, contributors can hide private projects from their own profile pages, but there is no global setting that repository owners can configure to do this.


* You could ask your contributors to manually remove the private repositories from their profile pages.
* You could create a separate organization for your private repositories, and then invite only the contributors who need access to those repositories to join the organization.

**Reply by froitag** [reply]
@nirav-gajera thanks the quick response.

Using a dedicated organization looks like the way to go when wanting to hide contributions from other people of your organization (contributions to private repos are anyways not visible to the general public).
